cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -122,6 +122,12 @@ def _write_chunk(request, courselike_key):
     course_dir = data_root / subdir
     filename = request.FILES['course-data'].name
 
+    def error_response(message, status):
+        """
+        Returns Json error response
+        """
+        return JsonResponse({'ErrMsg': message, 'Stage': -1}, status=status)
```

Comment:
What does `Stage` represent?

Reply:
Stage of error. ATM all errors have `-1` indicating the stage on which the error occurred. e.g. upload, unpacking, verifying, updating

Reply:
we can maybe have more specific statuses for error code within the different stages for more insight!

error in upload: status **-1**
error in unpacking: status **-2**
error in verifying: status **-3**
error in updating: status **-4**

Thought for future improvement, will help to identify the step where the error occurred when going through logs!

Reply:
This has already been implemented. -1 for unpacking, -2 for verifying and -3 for updating. There is no error status for the upload state at the moment which I think should be added here

Reply:
This PR is just for adding the logs, We have implement more when we work on specific task.